misc/loop/docker-compose.yml

```diff
@@ -0,0 +1,48 @@
+version: "3"
+services:
+  traefik:
+    image: "traefik:v2.10"
+    restart: unless-stopped
+    network_mode: host
+    container_name: "traefik"
+    command:
+      - "--api.insecure=true"
+      - "--providers.file=true"
+      - "--providers.file.watch=true"
+      - "--providers.file.directory=/etc/traefik/configs"
+      - "--entrypoints.web.address=:80"
+      - "--entrypoints.traefik.address=127.0.0.1:8080"
+    ports:
+      - "80:80"
+      - "8080:8080"
+    volumes:
+      - ./traefik:/etc/traefik/configs
+
+  gnoweb:
+    image: ghcr.io/gnolang/gno/gnoweb-slim
+    network_mode: host
+    restart: unless-stopped
+    entrypoint:
+      - gnoweb
+      - --bind=localhost:8888
+      - --remote=rpc.gno.land:80
+      - --views-dir=./gnoweb/views
+    extra_hosts:
+      - rpc.gno.land:127.0.0.1
```

Comment:
I would change this to `rpc.gno.local`. I saw in the browser the error message `gnoweb failed to query gnoland: server at 'http://rpc.gno.land:80' returned 502 Bad Gateway` and I thought that it was actually trying to connect to the live server